### PR TITLE
Fix indentation in precompile.jl

### PR DESCRIPTION
**Pull request by mkitti**
This deletes the two commented `end` statements and then reindents the `withenv` block.

### CODE REVIEW

**Comment by IanButterworth** [comment]
Might as well also run JuliaFormatter on the file. Note the outer let block isn't indented.

**Comment by mkitti** [comment]
I ran JuliaFormatter.jl on precompile.jl with default settings.

File: src/precompile.jl [hunk 1]
```diff
@@ -4,141 +4,195 @@ using Downloads
...
+        mkpath("registries/Registry/T/TestPkg")
+        write(
+            "registries/Registry/Registry.toml",
+            """
+name = "Registry"
+uuid = "37c07fec-e54c-4851-934c-2e3885e4053e"
+repo = "https://github.com/JuliaRegistries/Registry.git"
+[packages]
+$uuid = { name = "TestPkg", path = "T/TestPkg" }
+""",
```
Comment (IanButterworth) [inline src/precompile.jl:55]: These should be indented

File: src/precompile.jl [hunk 2]
```diff
@@ -4,141 +4,195 @@ using Downloads
...
+            Base.precompile(Tuple{typeof(Pkg.REPLMode.promptf)})
+            Base.precompile(Tuple{typeof(Pkg.REPLMode.repl_init),REPL.LineEditREPL})
+            Base.precompile(Tuple{typeof(Pkg.API.status)})
+            Base.precompile(
+                Tuple{
+                    typeof(Pkg.Types.read_project_compat),
+                    Base.Dict{String,Any},
+                    Pkg.Types.Project,
+                },
+            )
```
Comment (IanButterworth) [inline src/precompile.jl:159]: Lets leave these as one-liners.

Comment (mkitti) [reply src/precompile.jl:159]: Would you prefer that I do these manually or that I figure out a house style for Pkg.jl?

Comment (IanButterworth) [reply src/precompile.jl:159]: Just manually. I don't think we should go beyond this file

**Comment by mkitti** [comment]
Does this work or you do you want me to revert some of the altered lines?

**Comment by IanButterworth** [comment]
Thanks

**Pull request merged**